src/lammpsio/topology.py

```diff
@@ -1,3 +1,5 @@
+from collections.abc import MutableMapping
```

Comment:
I prefer not to import external classes into our namespace.

```suggestion
import collections.abc
```